src/foundry/client/pixi/extensions/rectangle.d.mts

```diff
@@ -47,7 +41,7 @@ declare module "pixi.js" {
      * @param point - A point to test for location relative to the rectangle
      * @returns Which edge zone does the point belong to?
      */
-    _getEdgeZone(point: Canvas.Point): PIXI.Rectangle["CS_ZONES"];
+    _getEdgeZone(point: Canvas.Point): PIXI.Rectangle.CS_ZONES;
```

Comment:
These were just straight up incorrect before weren't they, huh. Lol.

Reply:
Seems that way